compiler/src/dotty/tools/dotc/core/CheckRealizable.scala

```diff
@@ -107,12 +115,21 @@ class CheckRealizable(implicit ctx: Context) {
    *       also lead to base types with bad bounds).
```

Comment:
This comment should be updated.

Reply:
Actually, the comment is now correct. The previous implementation did less than the comment implied.